### DISCUSSION TITLE
tutorials/federation-typescript/08-using-directives

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#129] [course feedback]
# tutorials/federation-typescript/08-using-directives

https://www.apollographql.com/tutorials/federation-typescript/08-using-directives

**Comment by linomassarani** [comment]
There is a type mismatch that is causing an error. It that can be fixed by making `name` property optional like below: 

```ts
export type RecipeModel = {
  id: string;
  name?: string;
};
```

**Reply by lizhennessy** [reply]
Hi there @linomassarani ! I am not seeing an error in my terminal when running my project with the following `RecipeModel` definition. Could you share the error that you're seeing with me?

```
export type RecipeModel = {
  id: string;
  name: string;
};
```
